VocabularyTrainer/Home Screen/AboutViewController.swift

```diff
@@ -0,0 +1,194 @@
+                langString = "pt-BR"
+            } else {
+                langString = languageCode
+            }
+        }
+        return langString
+    }
+    /// The PP's location.
+    private var urlString: String {
+        "https://htmlpreview.github.io/?https://github.com/misteu/VocabularyTraining/blob/master/VocabularyTrainer/\(langCodeString).lproj/pp.html"
+    }
+
+    /// Close button for the navbar, shown when initial url (`urlString`) is loaded.
+    private lazy var closeButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(title: Strings.backButtonTitle,
+                                     image: UIImage(systemName: "x.circle"),
+                                     primaryAction: .init(handler: { [weak self] _ in
+            self?.dismiss(animated: true)
+        }), menu: nil)
+        button.tintColor = HomeViewModel.Colors.flippyGreen
+        return button
+    }()
+
+    /// Back button for the navbar, shown when navigated away from `urlString`.
+    private lazy var backButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(title: Strings.backButtonTitle,
+                                     image: UIImage(systemName: "chevron.left"),
+                                     primaryAction: .init(handler: { [weak self] _ in
+            self?.webView.goBack()
+            }), menu: nil)
+        button.tintColor = HomeViewModel.Colors.flippyGreen
+        return button
+    }()
+
+    /// Label showing the version number of the app.
+    private let versionLabel: UILabel = {
+        let label = UILabel()
+        let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
+        label.text = "FlippyLearn Version: \(appVersion ?? "unknown")"
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textAlignment = .center
+        label.backgroundColor = HomeViewModel.Colors.flippyGreen
+        label.textColor = .black
+        return label
+    }()
+
+    /// View below label making it nicer for devices without home button.
+    private let bottomView: UIView = {
+        let view = UIView()
+        view.backgroundColor = HomeViewModel.Colors.flippyGreen
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    // MARK: - Init
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        webView.navigationDelegate = self
+        setupUI()
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    // MARK: - View Lifecycle
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        navigateToPP()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        title = Strings.title
+        view.backgroundColor = .systemBackground
+        navigationItem.leftBarButtonItem = closeButton
+        webView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(webView)
```

Comment:
Maybe we could add an extension of UIView to add the views on array and with the translateAutoResizing on. What do you think?

Reply:
you mean inside of this extension?

![image](https://user-images.githubusercontent.com/35889530/213878321-41670dfe-89e6-4a86-bc2d-267499147897.png)

makes sense! updated that: cb106d5785ba24adb5dc47990be4b548812a9d14